### PR TITLE
(SIMP-2285) Create simp_nfs profile module

### PR DESCRIPTION
**Pull request by trevor-vaughan**
Initial cut of the simp_nfs profile module

SIMP-2285 #close

### CODE REVIEW

File: .fixtures.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+---
+fixtures:
+  repositories:
+    auditd: https://github.com/simp/pupmod-simp-auditd
+    augeasproviders_core:
+      repo: https://github.com/simp/augeasproviders_core
+      branch: simp-master
```
Comment (jeefberkey) [inline .fixtures.yml]: Try all of these on master

File: README.md [hunk 2]
```diff
@@ -1,2 +1,112 @@
...
+## Setup
+
+### What simp_nfs affects
+
+This module provides commonly used configurations for NFS server and client
+systems.
+
+## Usage
+
+### Serve NFS Home Directories over Stunnel
```
Comment (jeefberkey) [inline README.md]: Why does this start with stunnel?

Comment (trevor-vaughan) [inline README.md]: @jeefberkey Because that's the one that works with local logins. KRB5 isn't done yet.

File: metadata.json [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+    "simp"
+  ],
+  "dependencies": [
+    {
+      "name": "simp/nfs",
+      "version_requirement": ">= 6.0.0 < 7.0.0"
+    },
+    {
+      "name": "simp/simplib",
+      "version_requirement": ">= 2.1.0 < 3.0.0"
```
Comment (op-ct) [inline metadata.json]: :no_good_man:  Must be `>= 3.2.0 < 4.0.0`

File: spec/acceptance/nodesets/default.yml [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+      simp:
+        baseurl: https://packagecloud.io/simp-project/6_X_Alpha_Dependencies/el/7/$basearch
+        gpgkeys:
+          - https://raw.githubusercontent.com/NationalSecurityAgency/SIMP/master/GPGKEYS/RPM-GPG-KEY-SIMP
+      epel:
+        mirrorlist: 'https://mirrors.fedoraproject.org/metalink?repo=epel-7&arch=$basearch'
+        gpgkeys:
+          - https://getfedora.org/static/352C64E5.txt
+
+#  server6:
```
Comment (nick-markowski) [inline spec/acceptance/nodesets/default.yml]: Still in testing mode

**Pull request merged**